src/app/features/tag/store/tag.effects.ts

```diff
@@ -237,7 +237,8 @@ export class TagEffects {
     tap((arg) => console.log('Error INFO Today:', arg)),
     tap(({activeId, allTasks}) => {
       const allIds = allTasks.map(t => t && t.id);
-      const r = confirm('Nooo! We found some tasks with no data. It is strongly recommended to delete them to avoid further data corruption. Delete them now?');
+      const r = confirm('Nooo! We found some tasks with no data.\
```

Comment:
I don't think that max line length adds much value here. I would ignore this line.

Reply:
What about disabling the max-length rule only for strings? There is a configuration option available for that: https://eslint.org/docs/rules/max-len#ignorestrings

Reply:
Should be ok now, I disabled the max-length for strings